src/extension.ts

```diff
@@ -63,6 +64,8 @@ export async function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(new Reporter());
     setAssetContext(context);
 
+    registerUriHandler(context);
```

Comment:
Can you please leave some documentation on how this registerUriHandler works? if official docs are present ?  I found some in the vscode official docs is this the same ? 

https://code.visualstudio.com/api/references/vscode-api#UriHandler
![image](https://github.com/user-attachments/assets/c39fd414-6a40-403f-b18a-d89fe90bad9b)